Util/Xamarin.Kotlin.BindingSupport/native/build.gradle

```diff
@@ -1,6 +1,6 @@
 plugins {
     id 'java'
-    id 'org.jetbrains.kotlin.jvm' version '1.3.41'
+    id 'org.jetbrains.kotlin.jvm' version '1.3.0'
```

Comment:
Just a note about the version numbers: As I update the version to the later ones, it struggles to find various members. So, I just dropped it all the way down. Not sure what is happening here, but it works now...